README.adoc

```diff
@@ -0,0 +1,183 @@
+ifdef::env-github[]
+:status:
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:table-caption!:
+:namespace: mobile-security-service
+endif::[]
+
+:toc:
+:toc-placement!:
+
+= AeroGear UnifiedPush Server
+
+ifdef::status[]
+.*Project health*
+image:https://travis-ci.org/aerogear/aerogear-unifiedpush-server.png[Build Status (Trevis), link=https://travis-ci.org/aerogear/aerogear-unifiedpush-server]
+image:https://img.shields.io/:license-Apache2-blue.svg[License (License), link=http://www.apache.org/licenses/LICENSE-2.0]
+image:https://maven-badges.herokuapp.com/maven-central/org.jboss.aerogear.unifiedpush/unifiedpush-parent/badge.svg[Maven Central (Maven Central), link=https://maven-badges.herokuapp.com/maven-central/org.jboss.aerogear.unifiedpush/unifiedpush-parent]
+endif::[]
+
+:toc:
+toc::[]
+
+== Overview
+
+The _AeroGear UnifiedPush Server_ is a server that allows sending push notifications to different (mobile) platforms and has support for:
+* https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html#//apple_ref/doc/uid/TP40008194-CH8-SW1[Apple’s APNs (HTTP/2)]
+* https://firebase.google.com/docs/cloud-messaging/[Firebase Cloud Messaging]
+
+image::https://raw.githubusercontent.com/aerogear/aerogear-unifiedpush-server/master/ups-ui-screenshot.pnghttps://raw.githubusercontent.com/aerogear/aerogear-unifiedpush-server/master/ups-ui-screenshot.png[Create Server,align="center"]
+
+== Project Info
+
+|===
+| License:        | Apache License, Version 2.0
+| Build:          | https://maven.apache.org[Maven]
+| Documentation:  | https://docs.aerogear.org
+| Issue tracker:  | https://issues.jboss.org/browse/AEROGEAR
+| Mailing lists:  | https://groups.google.com/forum/#!forum/aerogear[aerogear-dev]
+|===
+
+== Prerequisites
+
+|===
+| https://maven.apache.org/install.html[Maven]
+| https://docs.docker.com/install/[Docker]
```

Comment:
@danielpassos @secondsun is it supporting any version? What are the minimum versions required?

Reply:
Nopz!